### PR TITLE
Item

### PR DESCRIPTION
**Pull request by SkylarSLiang**
-made a basic skeleton of Item.js component
-will make updates depending on other components

### CODE REVIEW

File: testing/src/components/container/Item.js [hunk 1]
```diff
@@ -5,6 +5,62 @@
 
 import React from "react";
 
-export const Item = () => {
- return <div>NewFile</div>;
+const MakeList = (props) => {
+
+    return props.items.map(items => 
+        <div key={items.id} style={{
+            border:"solid 1px black",
```
Comment (bilal-wsib) [inline testing/src/components/container/Item.js:12]: style in css not inline, it messes with cascading

**Comment by Aditya-wsib** [comment]
lgtm

File: testing/src/components/container/Item.js [hunk 2]
```diff
@@ -5,6 +5,62 @@
...
+            }}>
+                {items.content}
+            </p>
+            <button onClick={() => props.editItem()}>Edit</button>
+            <button onClick={() => props.deleteItem()}>Delete</button>
+        </div>
+    )
+}
+
+export const Item = ({ props }) => {
```
Comment (bilal-wsib) [inline testing/src/components/container/Item.js:27]: no need for { or even (

**Review by bilal-wsib — APPROVED** [review]
looks good

File: testing/src/components/container/Item.js (hunk 2 above)
Comment (SkylarSLiang) [reply testing/src/components/container/Item.js:27]: fixed!

File: testing/src/components/container/Item.js (hunk 1 above)
Comment (SkylarSLiang) [reply testing/src/components/container/Item.js:12]: fixed!

**Comment by SkylarSLiang** [comment]
Made a better version

**Pull request closed without merging**